app/views/users/new.html.erb

```diff
@@ -0,0 +1,13 @@
+<h1>Create an Account</h1>
+<form method="POST" action="/users" enctype="multipart/form-data">
+  <input type="hidden" name="authenticity_token" value="<%= "#{Rack::Protection::AuthenticityToken.token(env['rack.session'])}" %>" />
+  <label for="email">Email</label>
+  <input type="email" name="email" id="email" value="<%= @user.email %>">
+  <%= flash[:notice] %>
```

Comment:
I'm fairly confident you are double rendering the flash, since the layout also has a section for the flash messages.